unity-client/Assets/Scripts/MainScripts/DCL/Components/BuilderInWorld/DCLName.cs

```diff
@@ -58,7 +58,7 @@ public override IEnumerator ApplyChanges(string newJson)
         return null;
     }
 
-    public void ApplyNewValue(string newJson)
+    void ApplyNewValue(string newJson)
```

Comment:
Usually we explicitly set the access modifier but you dont have to put the `private` in all of them right now, you can do it progressively (the IDE might help you with this).